templates/partials/question.html

```diff
@@ -93,9 +93,11 @@
 }) %}
   {%- if content.list -%}
     {% set list = content.list %}
-    <div class="ons-u-mb-m">
-       {% include 'partials/summary/list-summary.html' %}
-    </div>
+    {%- if content.list.list_items | length > 0 -%}
```

Comment:
Not sure why you would have 2 separate conditions, I don't see the benefit of the set if there are no list_items. Empty list items are also considered false so you could just have this
```  
{%- if content.list and content.list.list_items -%}
   {% set list = content.list %}
    <div class="ons-u-mb-m">
        {% include 'partials/summary/list-summary.html' %}
    </div>
{% endif %}
```

Reply:
Yeah I agree, I've fixed that. I originally tried having one condition in a slightly different way and it fell over but that seems to work